### DISCUSSION TITLE
Cannot import several models with most recent version of package

### DISCUSSION THREAD

**Discussion by egolub22** [Nixtla/statsforecast#876] [Q&A] [answered]
I am trying to import a series of models but several of them are giving me issues. 

`Cell 1:

    from statsforecast.models import (
        AutoARIMA,
        AutoETS,
        AutoCES,
        AutoTheta,
        ARCH,
        Theta,
        OptimizedTheta,
        DynamicTheta,
        DynamicOptimizedTheta,
        MSTL,
        SimpleExponentialSmoothing,
        SimpleExponentialSmoothingOptimized,
        SeasonalExponentialSmoothing,
        SeasonalExponentialSmoothingOptimized,
        Holt,
        HoltWinters,
        SeasonalExponentialSmoothingOptimized)

`

Gives this error:

<img width="1587" alt="ImportError" src="https://github.com/user-attachments/assets/40d256a4-207b-42b2-8444-959e329480af">

Other models like GARCH and TBATS are also unable to be imported. I have tried this on versons 1.4.0, 1.7.5, and 1.7.6 and gotten the same errors

**Comment by jmoralez** [comment, marked as answer]
If you get the same errors on 1.7.5 then it's not really a problem with the latest version, you may have installation issues. Maybe you installed it with both conda and pip. Can you try uninstalling from both and installing again?

**Reply by egolub22** [reply]
Managed to get it working doing something similar. Working in a Sagemaker space that wipes the virtual env every time its closed. The .yml I was using to reinstall all the packages was not setup correctly
